### PR TITLE
specify installation prefix via $GEM_HOME in RubyGem easyblock

### PR DESCRIPTION
**Pull request by boegel**
fix for #646

### CODE REVIEW

File: easybuild/easyblocks/generic/rubygem.py [hunk 1]
```diff
@@ -81,4 +81,4 @@ def test_step(self):
 
     def install_step(self):
         """Install Ruby Gems using gem package manager"""
-        run_cmd('gem install --local %s' % self.ext_src)
+        run_cmd("gem install --install-dir %s --local %s" % (self.installdir, self.ext_src))
```
Comment (boegel) [inline easybuild/easyblocks/generic/rubygem.py]: no, wait, this ain't right...

`-i, -​-install-dir DIR - Gem repository directory to get installed gems`

so, `--install-dir` is supposed to point to `$EBROOTRUBY`?! :-/

**Comment by boegel** [comment]
@rjeschmi: please review

**Comment by wpoely86** [comment]
looks fine

**Comment by boegel** [comment]
Tested with https://github.com/hpcugent/easybuild-easyconfigs/pull/1889, good to go.

Thanks for the review @wpoely86!

**Pull request merged**